resources/ec2-network-interfaces.go

```diff
@@ -37,14 +37,12 @@ func ListEC2NetworkInterfaces(sess *session.Session) ([]Resource, error) {
 }
 
 func (e *EC2NetworkInterface) Remove() error {
+
 	if e.eni.Attachment != nil {
-		_, err := e.svc.DetachNetworkInterface(&ec2.DetachNetworkInterfaceInput{
+		e.svc.DetachNetworkInterface(&ec2.DetachNetworkInterfaceInput{
```

Comment:
Ignoring the error completely has the problem that it would assume the detachment worked properly on all other issues too (for example a connection issue).

Can we specifically check that the error indicates that the network interface is already gone?

Reply:
I added a check for the specific error and ignore it, otherwise if it is unexpected it will get returned back.